src/components/CurrencyInputPanel/index.tsx

```diff
@@ -151,18 +151,6 @@ const FiatRow = styled(LabelRow)<{ redesignFlag: boolean }>`
   height: ${({ redesignFlag }) => !redesignFlag && '24px'};
 `
 
-const NoBalanceState = styled.div`
```

Comment:
oh wait i think this stuff is needed for swap -- i totally forgot this component is also used in liquidity select

Reply:
discussed in DMs: only display on swap, not on liquidity context

Reply:
modified in latest commit to only remove in the add liquidity screen. this one shows what swap looks like: 
![Screen Shot 2022-08-18 at 11 19 12 PM](https://user-images.githubusercontent.com/41491154/185535702-b3f98c68-8412-4b9c-9221-a9de0f653105.png)